lib/cs_guide/resources/brand.ex

```diff
@@ -20,7 +20,7 @@ defmodule CsGuide.Resources.Brand do
   @doc false
   def changeset(brand, attrs) do
     brand
-    |> cast(attrs, [:name, :description, :deleted])
+    |> cast(attrs, [:name, :description, :deleted, :website, :logo, :member])
```

Comment:
`:website` is _good_ for now but should be _refactored_ to use the `Fields.url` once available: https://github.com/dwyl/fields/issues/16 which will perform validation. 👍